src/AppConstants.js

```diff
@@ -1,5 +1,5 @@
 // const APIRoot = "http://localhost:8000" // When running locally
-const APIRoot = "https://happy-youtube-watcher.herokuapp.com"
+const APIRoot = "https://hytw.hebronlab.com/"
```

Comment:
Will the slash at the end mess anything up? The old url didn't have a slash at the end